backend/src/main/java/com/example/demo/categories/CategoryService.java

```diff
@@ -23,4 +25,21 @@
 
     }
 
+    public List<Category> getCategories() {
+
+        return categoryRepository.findAll().stream()
```

Comment:
Let the database do the sorting.
```suggestion
        return categoryRepository.findAll(Sort.by("categoryName"));
```

Reply:
This is cool.